Symfony/CS/Tests/Fixer/PSR2/SwitchCaseSpaceFixerTest.php

```diff
@@ -0,0 +1,173 @@
+                '<?php
+                switch ($a) {
+                    case "prod" :
+                        break;
+                }
+                ',
+            ),
+            array(
+                '<?php
+                switch ($a) {
+                    case 42:
+                        break;
+                }
+                ',
+                '<?php
+                switch ($a) {
+                    case 42 :
+                        break;
+                }
+                ',
+            ),
+            array(
+                '<?php
+                switch ($a) {
+                    case false:
+                        break;
+                }
+                ',
+                '<?php
+                switch ($a) {
+                    case false :
+                        break;
+                }
+                ',
+            ),
+            array(
+                '<?php
+                switch ($a) {
+                    case 42:
+                        break;
+                }
+                ',
+                '<?php
+                switch ($a) {
+                    case 42    :
+                        break;
+                }
+                ',
+            ),
+            array(
+                '<?php
+                $a = 5.1;
+                $b = 1.0;
+                switch($a) {
+                    case (int) $a < 1: {
+                        echo "leave alone";
+                        break;
+                    }
+                    case ($a < 2)/* test */: {
+                        echo "fix 1";
+                        break;
+                    }
+                    case (3): {
+                        echo "fix 2";
+                        break;
+                    }
+                    case /**/(/**/ // test
+                        4
+                        /**/)//
+                        /**/: {
+                        echo "fix 3";
+                        break;
+                    }
+                    case (((int)$b) + 4.1): {
+                        echo "fix 4";
+                        break;
+                    }
+                    case ($b + 1) * 2: {
+                        echo "leave alone";
+                        break;
+                    }
+                }
+                ',
+                '<?php
+                $a = 5.1;
+                $b = 1.0;
+                switch($a) {
+                    case (int) $a < 1 : {
+                        echo "leave alone";
+                        break;
+                    }
+                    case ($a < 2)/* test */ : {
+                        echo "fix 1";
+                        break;
+                    }
+                    case (3) : {
+                        echo "fix 2";
+                        break;
+                    }
+                    case /**/(/**/ // test
+                        4
+                        /**/)//
+                        /**/ : {
```

Comment:
sounds good to me, it is what the fixer should do :)
(I opt the tests to make sure the fixer does not delete the comments and does not crash, so all good to me)